### PR TITLE
Add all current api calls to the docs

### PR DESCRIPTION
**Pull request by Hellowlol**
This document has been created from docs_md command. Its just to list whats currently available.
The docs are as good as plexpy own docs :(
I think it would be wise to switch to google style docs in the entire project as its much simpler to parse to something decent.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -23,58 +24,386 @@ Response example
...
+Get all details about a user from plextv
+
+
+### get_user_list
+
+
+### notify
+
+
+### random_arnold_quotes
```
Comment (drzoidberg33) [inline API.md:376]: The only good reason to have this api. :laughing:

Comment (Hellowlol) [inline API.md:376]: Well when all you need to do is slap decorator to add it to the api this is what happens... :P, while that being said, some stuff should be imported manually. I think i went a little overboard..

**Comment by JonnyWong16** [comment]
I want to go through and add documentation for everything first.

**Comment by Hellowlol** [comment]
Alright

**Pull request closed without merging**